src/utilities.ts

```diff
@@ -110,47 +112,29 @@ function pyCrc32IsAvailable(): boolean {
   return lastOutputLine == "AVAILABLE";
 }
 
-function CRCutilityIsAvailable() {
-  // check if the crc32 utility has been added to the the VectorCAST installation
 
-  // check the value of the option, I know this is extra work if
-  // the vcast installation is on the path but this is done once and it make the code cleaner
-  let settings = vscode.workspace.getConfiguration("vectorcastTestExplorer");
-  const installationLocation = settings.get("vcastInstallationLocation", "");
+function CRCutilityIsAvailable(vcastInstallationPath:string) {
```

Comment:
Maybe rename to `getCRCutilityPath` or something similar, since we are returning a path or undefined